runelite-client/src/main/java/net/runelite/client/plugins/chatleagueicons/ChatLeagueIconsConfig.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019, hsamoht <https://github.com/hsamoht>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.chatleagueicons;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("chatLeagueIcons")
+public interface ChatLeagueIconsConfig extends Config
+{
+	@ConfigItem(
+		keyName = "leagueIcon",
+		name = "Icon",
+		description = "The league icon to display in chat.",
+		position = 1
+	)
+	default LeagueIcon leagueIcon()
```

Comment:
I dont think the look of the icon should be configurable

Reply:
Honestly why has this so much configuration in general, just turn the plugin on or off.

Reply:
It has already been changed to a no-config plugin. We have also fixed the other issues mentioned in this PR. The latest version can be seen here https://github.com/Adam-/runelite/tree/leagueicons

Reply:
Ah, nice. @Adam- should really start making PRs with that stuff its kind of hard to track :/